packages/account/src/Controllers/ReportsController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+
+namespace Account\Controllers;
+
+
+use Account\Services\EmployeeSummaryService;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Illuminate\Support\Collection;
+
+class ReportsController
+{
+
+    public function employeeSummary(Request $request)
+    {
+        $dateFilter = $request->input('date_filter', null);
+        if ($dateFilter) {
+            [$d1, $d2] = explode(' - ', $dateFilter);
+            $d1 = Carbon::parse($d1)->startOfMonth();
```

Comment:
@KarolisNarkevicius isn't it dangerous to just "parse" date? What if someone enters "12-11-2019" - is it 12th November or 11th December?

Reply:
@PovilasKorop we could add some sort of validation to prevent invalid format input from the user. As for the parsing, carbon will prioritize 'd-m-Y' format in this case, so the first number will be the day.